pages/api/auth/[...nextauth].ts

```diff
@@ -11,7 +11,7 @@ export const authOptions = {
       tenantId: process.env.AZURE_AD_TENANT_ID || '',
       authorization: {
           params: {
-              scope: "openid User.Read User.ReadBasic.all offline_access",
+              scope: "openid User.Read offline_access",
```

Comment:
Was an older fix that I apparently never pushed. We've discussed this as an unnecessary permission before, though it wasn't causing the app approval auth issue that we were dealing with at the time.